### PR TITLE
Fix npm scripts on Windows

### PR DESCRIPTION
**Pull request by atomgomba**
This is a complement to #2182. Tested on W10.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,9 +7,13 @@
   "chromium-args": "--disable-features=nw2",
   "default_locale": "en",
   "scripts": {
-    "start": "NODE_ENV=development gulp debug",
+    "start": "npx run-script-os",
```
Comment (chmelevskij) [inline package.json:10]: Pretty sure you don't need `npx` when running package from `node_modules`

```
"start": "run-script-os"
```

Comment (atomgomba) [reply package.json]: Yes, seemingly it is not necessary. Thanks!

**Comment by TheIsotopes** [comment]
Working again on Windows 10 ... thx 👍

**Comment by McGiverGim** [comment]
I was fixing the same.. using cross-env https://www.npmjs.com/package/cross-env
I don't know which is better. Yours let us different scripts by environment. Mine simplifies it while we don't have different execution by environment.

Others thought?

**Comment by atomgomba** [comment]
If I understand correctly with `cross-env` you can execute the same command on different platforms and with `run-script-os` we have the *option* to execute different commands per platform. This is because `cross-env` only tries to solve the shell command syntax error when setting env vars in npm script while the other provides a more general solution for npm script platform cross-compatibility. For instance you can have different arguments for the same command or execute a different command depending on your platform.

**Comment by McGiverGim** [comment]
Exactly. This is the reason why I ask.
```json
  "scripts": {
    "start": "cross-env NODE_ENV=development gulp debug",
    "gulp": "gulp",
    "release": "cross-env NODE_ENV=production gulp release",
    "test": "karma start test/karma.conf.js"
  },
```

Currently we only have one command for all the environments, so we don't need to differentiate between them. So I don't know if better use the simplest way (cross-env) or the powerful one thinking in a future (script-os).

**Comment by McGiverGim** [comment]
Added a PR with `cross-env`. I was adding the vue devtools when I come across the problem. After we discuss about this, I can modify my solution without problem if we decide to go with `script-os`.

**Comment by atomgomba** [comment]
I feel like `run-script-os` is more future-proof as in less limited while being less complex (5 files, 11.4 kB) vs. `cross-env` (10 files, 27.4 kB). In other words it can do more with less code, but apart of this I don't really have strong reasons for or against this solution over the other solution. With `cross-env` the required changes are less obtrusive due to less lines modified. :shrug:

**Comment by McGiverGim** [comment]
Yes. Let's take the decision the person who does the merge, @mikeller what do you think? I'm ok with both approaches.

**Review by mikeller — APPROVED** [review]

**Comment by mikeller** [comment]
See https://github.com/betaflight/betaflight-configurator/pull/2227#issuecomment-709353719.

File: package.json [hunk 2]
```diff
@@ -101,6 +105,7 @@
     "rollup": "^2.28.2",
     "rollup-plugin-vue": "^5.*.*",
     "rpm-builder": "^1.2.1",
+    "run-script-os": "^1.1.1",
```
Comment (mikeller) [inline package.json:108]: You forgot to commit the changes to `yarn.lock` resulting from this change.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (atomgomba) [reply package.json:108]: Hm, I was using `npm`. Just pushed the lock file for both :)

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,11077 @@
+{
+  "name": "betaflight-configurator",
```
Comment (mikeller) [inline package-lock.json:2]: npm is not used in this project - please remove.

Also, please make sure you test everything with yarn before oipening pull requests - failing to do so will result in errors.

**Review by mikeller — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
`cross-env` seems more commonly used.

**Review by mikeller — APPROVED** [review]

**Pull request merged**